mockgen/mockgen.go

```diff
@@ -98,13 +99,28 @@ func main() {
 		g.srcPackage = flag.Arg(0)
 		g.srcInterfaces = flag.Arg(1)
 	}
+
+	if *mockNames != "" {
+		g.mockNames = parseMockNames(*mockNames)
+	}
 	if err := g.Generate(pkg, packageName); err != nil {
 		log.Fatalf("Failed generating mock: %v", err)
 	}
 	if _, err := dst.Write(g.Output()); err != nil {
 		log.Fatalf("Failed writing to destination: %v", err)
 	}
 }
+func parseMockNames(names string) map[string]string {
+	mocksMap := make(map[string]string)
+	for _, kv := range strings.Split(names, ",") {
+		parts := strings.SplitN(kv, "=", 2)
+		if len(parts) != 2 || parts[1] == "" {
+			log.Fatalf("bad mock names spec: %v", kv)
```

Comment:
If the parts are not OK you should continue with the next part and drop the current "bad" ones

Reply:
I would rather exit with error if that happens.

Reply:
Ok